### PR TITLE
i#6452,i#6536: Disable flaky attach_test, disable_test for unix x86-64

### PR DESCRIPTION
**Pull request by xdje42**
Adds two tests to the x86-64 unix ignore list:

i#6452 code_api|client.attach_test
i#6536 code_api|client.detach_test

Issue: #6452, #6536

### CODE REVIEW

File: suite/runsuite_wrapper.pl [hunk 1]
```diff
@@ -389,6 +389,8 @@
                 );
```
Comment (abhinav92003) [inline suite/runsuite_wrapper.pl:389]: Would be good to also mention x86-64 in the PR description and title.

**Review by abhinav92003 — APPROVED** [review]

File: suite/runsuite_wrapper.pl (hunk 1 above)
Comment (xdje42) [reply suite/runsuite_wrapper.pl:389]: These tests have also been seen to fail on aarch64 though.
Aren't references to the issues enough?

Comment (abhinav92003) [reply suite/runsuite_wrapper.pl:389]: Well, since this PR is disabling them for a specific suite, it makes sense to mention the suite in the description.

Comment (derekbruening) [reply suite/runsuite_wrapper.pl:389]: Could you point to an aarch64 failure?  Prior to the ptrace fix they failed due to permissions; after that fix they have never failed AFAIK.  I logged into that test machine and ran each test 200x in a row w/ no failures manually as I noted in the issue/PR.

Comment (derekbruening) [reply suite/runsuite_wrapper.pl:389]: > Adds two tests to the unix ignore list:

I'm agreeing with @abhinav92003  -- there is no ignore list for all of "unix": this PR is adding these only to the x86_64 Linux list.

Comment (xdje42) [reply suite/runsuite_wrapper.pl:389]: Done

**Pull request merged**